packages/meditrak-app/app/sync/Synchroniser.js

```diff
@@ -403,4 +406,15 @@ export class Synchroniser {
       await migrationMethods[m](this, setProgressMessage);
     }
   }
+
+  async cleanupChangesAfterPush(changes) {
+    for (const change of changes) {
+      try {
+        await change.cleanupAfterPush(this.database);
+      } catch (e) {
+        // TODO: log
+        console.error(`Error thrown in cleanupAfterPush: ${e}`);
```

Comment:
I've noticed a few of these console.log calls around. I'm assuming that they're being left in intentionally?

Reply:
These ones are, yeah. I'm not sure how to hook into bugsnag other than crashing the whole app so I just left these as console.logs with a TODO. Seems like the TODO comment could be clearer?